src/sign-in-xplat/README.md

```diff
@@ -0,0 +1,113 @@
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- csharp
+page_type: sample
+name: ".NET MAUI Multi-platform App Sign-in user"
+description: "This is a .NET MAUI Multi-platform App Sign-in user that sign-in users. The code in this sample is used by one or more articles on docs.microsoft.com."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-csharp
+---
+# .NET MAUI Multi-platform App - Sign-in user | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This sample demonstrates a .NET MAUI Multi-platform App that sign-in users by using the `Microsoft.Identity.Client`.
+
+<!-- IMAGE or CONSOLE OUTPUT of running/executed app -->
```

Comment:
Need output example.

Reply:
I got you @mmacy 

done | addressed from [bd39b](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/bd39b219b802630712d79645faa25bb239eaade2)